browser-test/utils.js

```diff
@@ -34,6 +34,12 @@ export const getElementByXPath = async (driver, xpath) => {
   return el;
 };
 
+export const getElementByCss = async (driver, css) => {
```

Comment:
@codesankalp let's use css instead of xpath which is more universally known

Reply:
Added it in this https://github.com/openwisp/openwisp-wifi-login-pages/pull/284